### DISCUSSION TITLE
posts/practical_implementation_of_mis_in_bidirectional_path_tracing/

### DISCUSSION THREAD

**Discussion by giscus** [JiayinCao/AGraphicsGuyNotes#14] [Giscus-Comments]
# posts/practical_implementation_of_mis_in_bidirectional_path_tracing/

In my previous post, I talked some basic stuff about naive bidirectional path tracing. However it is hard to show any real value since there are always too much noise comparing with best solutions depending on the scene to be rendered. That is because the contribution of each specific path is not properly weighted. And multiple importance sampling can be the key to the issue, the following comparison shows big difference between different methods.

https://agraphicsguynotes.com/posts/practical_implementation_of_mis_in_bidirectional_path_tracing/

**Comment by kingofthebongo2008** [comment]
Hello, equation 19 is not readable on chrome.
There is some typo somewhere i guess

**Reply by JiayinCao** [reply]
This happened to me in another post as well, I never got the chance to do a full check for all my blog posts.
If you are intereted in learning what causes this, take a look at this [conversation](https://discourse.gohugo.io/t/latex-not-working-in-newer-version-of-hugo/41786).

In a nutshell, hugo had a bug that allowed me to display latex code like this so that I never found out something wrong. There is nothing wrong with the latex code here. But in order to properly renderer it, it has to be within a shortcode or a code block, which I never did.

I'll do a full check of all math in my blog when I find times.
Thanks for letting me know about this. I think this particular issues should have already been addressed just now.

**Comment by kingofthebongo2008** [comment]
w_{i}=\overleftarrow{p_i}(\bar{y})(\underbrace{\dfrac{w_{i-1}}{\overrightarrow{p_i}(\bar{y})}}{\overleftarrow{p{\sigma,i-1}} \boxed{vc_i}}+\underbrace{\dfrac{1}{\overrightarrow{p_i}(\bar{y})}}_{\boxed{vcm_i}})\ \ \ \ (19)$
